pkg/rbd/rbd_util.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/klog"
+	"k8s.io/cloud-provider/volume/helpers"
```

Comment:
This new import will fail testing. Go prefers grouped imports to be sorted, `klog` and `helpers` should be switched.

These things can now be easier caught on developers system with commands like:

    make containerized-test TARGET=static-check

The container for testing will be built once, and re-used for new testing, Drop the `TARGET=static-check` to have all tests run (well, after #964).

Reply:
Thanks for the pointer, containerized tests are awesome!
I was struggling locally with an plain Alpine container and installing the deps myself.

Might be nice to add something about testing locally to the [development-guide.md](https://github.com/ceph/ceph-csi/blob/master/docs/development-guide.md)?